### PR TITLE
Renames and cleans up additionalert module

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Cleans up some of the code in the `additionalert` module and renames it to `modular_ert`.

Removes some unused `.dmi`s and one unused `.ogg`.

## Changelog

No player-facing changes.

### CODE REVIEW

File: modular_skyrat/modules/modular_ert/code/pizza/pizza_outfit.dm [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /datum/outfit/centcom/ert/pizza //da pizza for you and me
 	name = "Pizza Delivery Boy"
-
 	id = /obj/item/card/id/advanced/centcom/ert
-	suit =  /obj/item/clothing/suit/pizzahoodie
+	suit = /obj/item/clothing/suit/pizzahoodie
```
Comment (tf-4) [inline modular_skyrat/modules/modular_ert/code/pizza/pizza_outfit.dm]:
```suggestion
	suit = /obj/item/clothing/suit/toggle/jacket/hoodie/pizza
```
Verified ✔️

File: modular_skyrat/modules/modular_ert/code/pizza/weaponry.dm [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/obj/item/knife/hotknife
+	name = "thousand degree knife"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_ert/code/pizza/weaponry.dm:2]: Shouldn't this be `thousand degrees knife`? I'm actually not sure which is the correct way to spell it.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/modular_ert/code/pizza/weaponry.dm (hunk 2 above)
Comment (tf-4) [reply modular_skyrat/modules/modular_ert/code/pizza/weaponry.dm:2]: No

**Comment by GoldenAlpharex** [comment]
Ok then

**Pull request merged**